### DISCUSSION TITLE
Hi there!

### DISCUSSION THREAD

**Discussion by curiousdro1d** [ravynsoft/ravynos#410] [General]
This Project looks promising...

But i have just a few dumb questions, before i dig deeper into alternate OS.
I'm a win 10 user but used to be an mac os user too. I have a few mac os software license bought and they're unused now. at the moment i'm not happy with the windows ecosystem and neither with apple. sooo can you run software from the 2020s at this stage of this OS? I would hopp on it if it's stable enough to use it as an everyday system. i need it for special writing software, a bit of gaming and graphic-design. could you run Scrivener or Affinity Photo on this one? Best wishes!

**Comment by Anvith00** [comment]
> can you run software from the 2020s at this stage of this OS?

Unfortunately, no. The OS is in pre-release state, meaning this is way before it reaches alpha or an early-access release. There's a lot of work to be done, and the team looks for more active able volunteers (developers) to this project.
Specifically, on getting it to render Wayland with vmwgfx (the VMware graphics acceleration driver for Linux, and FreeBSD too, in this case).

There's other works to be done too, and the goal is to reach usability and appearance equivalent to macOS 10.15 (Catalina, released 2019).

**Reply by curiousdro1d** [reply]
Thanks for the fast reply! I hope at some day, there will be more OS to compete with the big players! But glad there will be options! Keep going anyways!
